config/webpack.prod.js

```diff
@@ -5,7 +5,7 @@ var commonConfig = require('./webpack.common.js');
 var helpers = require('./helpers');
 
 const ENV = process.env.NODE_ENV = process.env.ENV = 'production';
-const LAUNCHER_BACKEND_URL = process.env.LAUNCHER_BACKEND_URL || '/launch/api';
+const LAUNCHER_BACKEND_URL = process.env.LAUNCHER_BACKEND_URL;
```

Comment:
I think Minishift installations do not specify this parameter, but I may be wrong

Reply:
where is the minishift setup?

Reply:
so I'll make it optional then

Reply:
Try installing the openshift template and see if this parameter is set in the frontend deploymentconfig